packages/react-components/review/stable/react-components.api.md

```diff
@@ -1188,6 +1188,7 @@ export const _RemoteVideoTile: React_2.MemoExoticComponent<(props: {
     strings?: VideoGalleryStrings | undefined;
     participantState?: ParticipantState | undefined;
     showRemoteVideoTileContextualMenu?: boolean | undefined;
+    drawerMenuHostId?: string | undefined;
```

Comment:
Should this prop be in stable as well?

Reply:
Yes. It's an internal component.
If it was being exported by communication-react.api.md, that would be a problem